client/reader/user-stream/index.tsx

```diff
@@ -72,9 +97,20 @@ const UserStream = ( { streamKey, userId }: UserStreamProps ) => {
 		}
 	};
 
+	useEffect( () => {
+		fetchUser( userId ).then( ( data ) => {
+			const userData = {
+				...data,
+				has_avatar: true,
```

Comment:
I'm forcing `has_avatar` to `true`, similar to how it's handled here for the ReaderPostCard. Does this seem right? https://github.com/Automattic/wp-calypso/blob/ec5a2aeee1a07395d6411392f23f72c1720c1178/client/components/seo/reader-preview/index.js#L11-L20